src/_includes/components/footer.njk

```diff
@@ -31,24 +31,36 @@
 
       <script>
         const form = document.getElementById("newsletter-form");
+          const success = form.querySelector(".form-success");
 
         form.addEventListener("submit", (e) => {
           e.preventDefault();
           const url = form.getAttribute("action");
           const body = new URLSearchParams(new FormData(form)).toString();
-          const success = form.querySelector(".form-success");
-
+          console.log(body)
```

Comment:
Sorry, I wasn't clear on this. We only want to `console.log` in the Lambda function not here in client-side JS. It's helpful to log in the Function because we can access those logs later to diagnose issues. This we can't access because it's only the users' browser.

Reply:
Fixed! This was my bad. Left this in here from debugging.